### ISSUE TITLE
`--disable-uncorrectable` breaks semantically `%i[]` syntax

### ISSUE THREAD

**Issue by ogawa65a** [rubocop/rubocop#14353]
## Expected behavior

Custom cop `Rails/AvoidIncludes` is expected to replace `includes` with `preload` or `eager_load` in the following code:

```ruby
#rubocop/cops/rails/avoid_includes.rb
module RuboCop
  module Cop
    module Rails
      class AvoidIncludes < Base
        extend AutoCorrector

        MSG = 'Avoid using `includes`, use `preload` or `eager_load` instead.'.freeze

        def_node_matcher :includes_method?, <<~PATTERN
          (send _ :includes ...)
        PATTERN

        def on_send(node)
          return unless includes_method?(node)

          add_offense(node.loc.selector, message: MSG)
        end
      end
    end
  end
end
```

```ruby
#sample.rb
SomeModel
  .includes(org: %i[a
                    b
                    c])
```

```shell
rubocop --only Rails/AvoidIncludes --autocorrect --disable-uncorrectable sample.rb
```

Semantically same code with rubocop:todo comments is expected to be produced

## Actual behavior

Actually produced code is:

```ruby
#sample.rb
SomeModel
  # rubocop:todo Rails/AvoidIncludes
  .includes(org: %i[a # rubocop:todo Rails/AvoidIncludes # rubocop:todo Rails/AvoidIncludes
  # rubocop:enable Rails/AvoidIncludes
                    b
                    c])
```

This is not semantically same as the original code, because in `%i[a # ...]`, `#` is treated as a part of the symbol, not as a comment.
Furthermore, the `# rubocop:todo Rails/AvoidIncludes` comments are inserted third times.

## Steps to reproduce the problem

Use the codes described in the "Expected behavior" section.

## RuboCop version

```shell
$ rubocop -V                                                                              
1.74.0 (using Parser 3.3.8.0, rubocop-ast 1.42.0, analyzing as Ruby 3.4, running on ruby 3.4.4) [arm64-darwin24]
  - rubocop-performance 1.24.0
  - rubocop-rails 2.30.3
  - rubocop-rspec 3.6.0
```

**Comment by ogawa65a** [comment]
This may be duplicate of
- https://github.com/rubocop/rubocop/issues/14050

Now in the process of confirming...

**Comment by ogawa65a** [comment]
This issue was resolved in v1.75.8

- https://github.com/rubocop/rubocop/pull/14206
